science/gmt4/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 
 name                gmt4
-version             4.5.15
+version             4.5.17
 revision            1
```

Comment:
The `revision` line should be deleted because it starts at 0 when increasing the version and that is the default value.

Reply:
Also, please reference the Trac ticket in the commit message. Add this line to the end:

Closes: https://trac.macports.org/ticket/56433